test/controllers/github-tab-controller.test.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
we didn't have any tests for github-tab-controller before?  yikes!  Thanks for adding some.

Reply:
Not a one 😅 

I was tempted to get full coverage on it here, but I decided not to because "refactoring it apart into `GitHubTabItem` / `GitHubTabContainer` / `GitHubTabController`" is on my radar at some point, so I figured I'd backfill the rest then.